plugins/routed-eni/cni.go

```diff
@@ -324,19 +323,15 @@ func del(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 }
 
 func main() {
-	logger.SetupLogger(logger.GetLogFileLocation(defaultLogFilePath))
```

Comment:
You need to add in functionality to allow passing the log filepath (the default for this binary is "/var/log/aws-routed-eni/plugin.log", not "test.log") and we need to be sure that for this patch to merge, the same logging behaviour and configuration as the existing default is provided.